### PR TITLE
updated the files based on review from microsoft

### PR DESCRIPTION
**Pull request by InEightDevOps**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

File: certified-connectors/InEight/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,3049 @@
...
+          "text/plain"
+        ],
+        "produces": [
+          "application/json",
+          "text/plain"
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {}
```
Comment (sriyen-msft) [inline certified-connectors/InEight/apiDefinition.swagger.json:1436]: @InEightDevOps Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger.

File: certified-connectors/InEight/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,3049 @@
...
+    {
+      "oauth2_auth": []
+    }
+  ],
+  "paths": {
+    "/api/DailyPlan": {
+      "get": {
+        "description": "The Daily Plan integration allows customers to provide time keeping information for employees and equipment as recorded in InEight Plan. This integration includes summary information about the daily plan, from employee hours to cost items, equipment hours to cost items, payroll indicators, and notes. <br/>The Daily Plan integration can also be used to track and provide plan status in systems outside of InEight cloud platform. When a daily plan is used for this purpose, an optional response message can be used to trigger a status change in the daily plan to indicate the change has been successfully received by the external system.",
+        "operationId": "DailyPlan_Get",
+        "summary": "DailyPlan_Get",
```
Comment (sriyen-msft) [inline certified-connectors/InEight/apiDefinition.swagger.json:51]: @InEightDevOps Summary values are visible to the public and these values must be free of grammatical and spelling errors.

Docs - [https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions)

File: certified-connectors/InEight/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,3049 @@
...
+                    "type": "string",
+                    "description": "VendorCode"
+                  },
+                  "SourceSystemName": {
+                    "type": "string",
+                    "description": "SourceSystemName"
+                  },
+                  "SourceSystemId": {
+                    "type": "string",
+                    "description": "SourceSystemId"
```
Comment (sriyen-msft) [inline certified-connectors/InEight/apiDefinition.swagger.json:2871]: @InEightDevOps All the description values of properties are expected to be free from grammatical and spell errors. Please provide valid description values.

Docs - [Define operation and parameter summaries and descriptions](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions)

File: certified-connectors/InEight/apiProperties.json [hunk 4]
```diff
@@ -0,0 +1,35 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "CLIENTID",
```
Comment (sriyen-msft) [inline certified-connectors/InEight/apiProperties.json:8]: @InEightDevOps Can you please make it to `"[[DUMMY]]"` to maintain consistency with other submissions.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

**Comment by sriyen-msft** [comment]
@InEightDevOps I'm just checking in to see how things are going with updating your connector artifacts as per the given feedback. Please feel free to reach out with any questions you may have, Thanks.

**Comment by InEightDevOps** [comment]
Hi Srikant,

Dev team is working as per the feedback. We will let you know once we update our changes.


Riyaz Mohammed
DevOps Engineer IV
m +1 (650) 645 0526

**Comment by InEightDevOps** [comment]
Hi Srikanth,

We have pushed the changes to the Microsoft Dev branch as per your review comments. Do we need to create a new PR or the changes will populate automatically?




Riyaz Mohammed
DevOps Engineer IV
m +1 (650) 645 0526

From: Riyaz Mohammed
Sent: Monday, December 6, 2021 6:43 PM
To: microsoft/PowerPlatformConnectors ***@***.***>; microsoft/PowerPlatformConnectors ***@***.***>
Cc: DevOpsMail ***@***.***>; Mention ***@***.***>
Subject: RE: [microsoft/PowerPlatformConnectors] updated the files based on review from microsoft (PR #1280)

Hi Srikant,

Dev team is working as per the feedback. We will let you know once we update our changes.


Riyaz Mohammed
DevOps Engineer IV
m +1 (650) 645 0526

**Review by sriyen-msft — APPROVED** [review]

**Comment by sriyen-msft** [comment]
@InEightDevOps Apologies for the delay in response, We have a high number of submissions and had to prioritize the connectors which were ready for production deployment. We are reviewing submissions one by one sorted by created date time.

**Pull request merged**